GateMaterials.db

```diff
@@ -476,16 +664,241 @@ Testis: d=1.04 g/cm3 ; n=9
+        +el: name=Nitrogen   ; f=0.02866
+        +el: name=Oxygen     ; f=0.75752
+        +el: name=Sodium     ; f=0.00184
+        +el: name=Magnesium  ; f=0.00007
+        +el: name=Silicon    ; f=0.00006
+        +el: name=Phosphor   ; f=0.00080
+        +el: name=Sulfur     ; f=0.00225
+        +el: name=Chlorine   ; f=0.00266
+        +el: name=Potassium  ; f=0.00194
+        +el: name=Calcium    ; f=0.00009
+        +el: name=Iron       ; f=0.00037
+        +el: name=Zinc       ; f=0.00001
+        +el: name=Rubidium   ; f=0.00001
+
+Bones: d=1.4 g/cm3 ; n=18
+        +el: name=Hydrogen   ; f=0.07337
+        +el: name=Carbon     ; f=0.25475
+        +el: name=Nitrogen   ; f=0.03057
+        +el: name=Oxygen     ; f=0.47893
+        +el: name=Fluorine   ; f=0.00025
+        +el: name=Sodium     ; f=0.00326
+        +el: name=Magnesium  ; f=0.00112
+        +el: name=Silicon    ; f=0.00002
+        +el: name=Phosphor   ; f=0.05095
+        +el: name=Sulfur     ; f=0.00173
+        +el: name=Chlorine   ; f=0.00143
+        +el: name=Potassium  ; f=0.00153
+        +el: name=Calcium    ; f=0.10190
+        +el: name=Iron       ; f=0.00008
+        +el: name=Zinc       ; f=0.00005
+        +el: name=Rubidium   ; f=0.00002
+        +el: name=Strontium  ; f=0.00003
+        +el: name=Lead       ; f=0.00001
+
+PE:  d=0.93 g/cm3 ; n=2
+        +el: name=Hydrogen   ; n=2
+        +el: name=Carbon     ; n=1
+
+BoneEquiv: d=1.89 g/cm3 ; n=6
+        +el: name=Hydrogen   ; f=0.0310
+        +el: name=Carbon     ; f=0.3126
+        +el: name=Nitrogen   ; f=0.0099
+        +el: name=Oxygen     ; f=0.3757
+        +el: name=Chlorine   ; f=0.0005
+        +el: name=Calcium    ; f=0.2703
+
+MuscleEquiv: d=1.00 g/cm3 ; n=6
+        +el: name=Hydrogen   ; f=0.0841
+        +el: name=Carbon     ; f=0.6797
+        +el: name=Nitrogen   ; f=0.0227
+        +el: name=Oxygen     ; f=0.1887
+        +el: name=Chlorine   ; f=0.0013
+        +el: name=Calcium    ; f=0.0235
+
+LungEquiv: d=0.30 g/cm3 ; n=7
+        +el: name=Hydrogen   ; f=0.0836
+        +el: name=Carbon     ; f=0.6041
+        +el: name=Nitrogen   ; f=0.0167
+        +el: name=Oxygen     ; f=0.1733
+        +el: name=Magnesium  ; f=0.1136
+        +el: name=Silicon    ; f=0.0072
+        +el: name=Chlorine   ; f=0.0015
+
+CdTe:   d=5.85 g/cm3 ; n=2; state=solid
+        +el: name=Cadmium    ; f=0.468358
+        +el: name=Tellurium  ; f=0.531642
+
+PMMA:   d=1.195 g/cm3; n=3 ; state=solid
         +el: name=Hydrogen   ; f=0.080541
         +el: name=Carbon     ; f=0.599846
         +el: name=Oxygen     ; f=0.319613
 
-Epoxy:  d=1.0 g/cm3; n=3; state=Solid
+PMMA2:   d=1.19 g/cm3 ; n=3 ; state=solid
+        +el: name=Hydrogen   ; f=0.080538
+        +el: name=Carbon     ; f=0.599848
+        +el: name=Oxygen     ; f=0.319614
+
+PMMA3:   d=1.195 g/cm3; n=3 ; state=solid
+        +el: name=Hydrogen   ; f=0.080541
+        +el: name=Carbon     ; f=0.599846
+        +el: name=Sulfur     ; f=0.319613
```

Comment:
I would chose only one PMMA

Reply:
Yes, and I would go for PMMA2 as defined in NIST:
https://physics.nist.gov/cgi-bin/Star/compos.pl?refer=ap&matno=223

Reply:
kept only PMMA2 (and renamed it to PMMA)